src/borg/helpers/parseformat.py

```diff
@@ -688,7 +688,8 @@ class ItemFormatter(BaseFormatter):
         ('size', 'csize', 'dsize', 'dcsize', 'num_chunks', 'unique_chunks'),
         ('mtime', 'ctime', 'atime', 'isomtime', 'isoctime', 'isoatime'),
         tuple(sorted(hash_algorithms)),
-        tuple(('chunk_ids_%s' % alg for alg in sorted(hash_algorithms))),
+        tuple(['chunk_ids_%s' % alg for alg in sorted(hash_algorithms)] + [
+            'chunker_params_%s' % alg for alg in sorted(hash_algorithms)]),
```

Comment:
the chunker params are same within 1 archive, so guess you do not want to show them for each file?

Reply:
hmm, true, but this would make parsing the output of list harder. As it is now, you can just compare (item['chunk_ids_checksum'], item['chunker_params']) after parsing the output for each item.